k8s/operators/src/diskpool/main.rs

```diff
@@ -838,12 +814,6 @@ async fn main() -> anyhow::Result<()> {
                 .env("JAEGER_ENDPOINT")
                 .help("enable open telemetry and forward to jaeger"),
         )
-        .arg(
```

Comment:
Don't remove this flag from CLI as it would be breaking change, instead simply not use it.

Reply:
Alright

Reply:
Done